addons/l10n_it_edi/models/account_move.py

```diff
@@ -84,6 +84,7 @@ class AccountMove(models.Model):
         compute=lambda self: self._compute_linked_attachment_id('l10n_it_edi_attachment_id', 'l10n_it_edi_attachment_file'),
         depends=['l10n_it_edi_attachment_file'],
     )
+    l10n_it_edi_proxy_mode = fields.Char(compute="_compute_l10n_it_edi_proxy_mode")
```

Comment:
If we're adding this only to get a "proxy field" to the edi_mode, shouldn't it be a related field, instead of a compute?

Otherwise, we could make it a stored field that's saved on post, so we know what mode was used when posting... maybe it's out of the scope, or not deemed useful, but it's an idea.